### PR TITLE
RIA-3218 Legal rep apply for a late remission

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3218](https://tools.hmcts.net/jira/browse/RIA-3218)

### Change description ###
Legal rep apply for a late remission

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/features/apply-for-late-remission.feature [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Then I am on the `Check your answers` page
+    And I should see `My client has a remission, e.g. Asylum support, Legal Aid, Home Office waiver, Section 17/20` for the `Choose one of the following statements` answer
+    And I should see `20/02/1234` for the `Asylum Support reference number` answer
+    And I should see `Evidence1.pdf` in the `Asylum Support confirmation letter` field
+    And I click the `Submit` button
+
+    Then I should see the text `Your remission request has been submitted`
+    And I should see the text `What happens next`
+    And I should see the text `The Tribunal will review the remission request. The decision details will be available to`
+    And I should see the text `view in the appeal tab when the decision has bee made.`
```
Comment (divisathyan) [inline e2e/features/apply-for-late-remission.feature:45]: Typo for "has been made"

**Review by divisathyan — CHANGES_REQUESTED** [review]
One small typo change

File: e2e/features/apply-for-late-remission.feature (hunk 1 above)
Comment (tarun-palisetty) [reply e2e/features/apply-for-late-remission.feature]: Fixed the typo

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Review by divisathyan — APPROVED** [review]

**Pull request merged**